microshift_install/microshift-install-rpm.adoc

```diff
@@ -46,4 +46,8 @@ include::modules/microshift-accessing.adoc[leveloffset=+1]
 
 include::modules/microshift-accessing-cluster-locally.adoc[leveloffset=+2]
 
+include::modules/microshift-accessing.adoc[leveloffset=+1]
+
+include::modules/microshift-accessing-cluster-locally.adoc[leveloffset=+2]
+
```

Comment:
These lines duplicate those above.

Reply:
@fzdarsky I'm working on it. Merge conflicts are resolved, but it created some other errors that I am fixing.